extras/http_client_ota/http_client_ota.c

```diff
@@ -68,17 +68,23 @@ static unsigned int ota_firmaware_dowload_callback(char *buf, uint16_t size)
         return -1;
     }
 
-    // Ready for flash device, the erase NANDFLASH Block
-    if (flash_offset % SECTOR_SIZE == 0) {
-        unsigned int sector;
-
-        sector = flash_offset / SECTOR_SIZE;
-        sdk_spi_flash_erase_sector(sector);
```

Comment:
Why did you remove call to sdk_spi_flash_erase_sector()? It's nesessary to erase flash page before writing.

Reply:
Hmm, in my case, it is actually working without. I will try tomorrow to put it back and update the PR.

Reply:
Try to write a with customized firmware a page of 0.
Then write your firmware it without erase.
This must show you that erase is required. If it works without we got a problem.
Because flash does not work without erase.